### DISCUSSION TITLE
Postal incoming message http routes not working

### DISCUSSION THREAD

**Discussion by fiantinangon** [postalserver/postal#1658] [Help with using Postal]
<img width="622" alt="Screen Shot 2021-10-26 at 08 45 01" src="https://user-images.githubusercontent.com/9161278/138795272-e0c7ad5a-4246-4c04-88db-11bc38f1783a.png">
<img width="1231" alt="Screen Shot 2021-10-26 at 08 44 49" src="https://user-images.githubusercontent.com/9161278/138795275-10839725-c998-43e2-9e73-668c94bd3550.png">
Hi, i've set postal behind NGINX and finished setup all DNS configuration, but can't receive any email to my HTTP endpoint, but if I tested simulated incoming message using postal portal it's working.

But never receive any incomming message if I reply on real email from any email provider.

Do I need to add extra configuration on my NGINX  to allow receive incoming email and route to my HTTP endpoint ?
<img width="606" alt="Screen Shot 2021-10-26 at 08 19 12" src="https://user-images.githubusercontent.com/9161278/138792571-de69e440-d1df-4738-902b-5858193056f7.png">

**Comment by willpower232** [comment]
do you mean that Postal is unable to send any email to your route or that you can't give email to Postal via the api?

**Reply by fiantinangon** [reply]
@willpower232 i got error everytime trying to sending email to my routes via gmail.
<img width="797" alt="Screen Shot 2021-10-28 at 21 42 19" src="https://user-images.githubusercontent.com/9161278/139279403-1805c891-a3c3-4993-ad86-8703eb0da32f.png">

But i can simulate incoming message via postal, all incoming message successfully send to my routes.

But never receive incoming message from gmail.

this is my mx toolbox test result :

<img width="1109" alt="Screen Shot 2021-10-28 at 22 42 59" src="https://user-images.githubusercontent.com/9161278/139290198-d1b92bb5-cf60-4372-9189-ea1e4b18d4d4.png">

**Reply by willpower232** [reply]
how have you configured gmail to send email via Postal?

**Reply by fiantinangon** [reply]
> how have you configured gmail to send email via Postal?

Yes, well configured and I can send email through postal dashboard, the only issue is I cant receive incoming message 
![Uploading Screen Shot 2021-10-29 at 11.02.42.png…]()

**Reply by willpower232** [reply]
Unfortunately the screenshot did not fully upload, can you try again?

**Comment by ghost** [comment]
I also experienced the same issue @willpower232 
![image](https://user-images.githubusercontent.com/56396555/139469517-b1c8f222-cf02-45fb-a37a-0bc627970a45.png)

**Reply by NielsOsnabrugge** [reply]
having the same issue atm. @shirakawa-poi or @fiantinangon , did you manage to solve it?
